docker-compose.yml

```diff
@@ -200,6 +202,8 @@ services:
       - "2"
       - --rollup-ids
       - "3"
+      - --metrics-ip-port-address
+      - "0.0.0.0:9090"
```

Comment:
This could just be a `127.0.0.1`/`localhost`

Reply:
Also, the same config should be added to the operator setup

Reply:
localhost - no, since SocketAddr accepts only [ip]:[port] format. regarding 127.0.0.1 not sure it is a loopback so I will check it out

Reply:
Oh well, this is not convenient at all. Seen some people doing custom `value_parser`s for this. In our case doesn't really matter. Sure, 127.0.0.1 is a loopback.

Reply:
Since this is internal for the container and the port is only exposed, doesn't really matter, closing this